### ISSUE TITLE
update cosmos/ledger-go after #4 merged

### ISSUE THREAD

**Issue by okwme** [cosmos/gaia#1576]
## Summary of Bug

after temp fix in #1575 we should make permanent fix once https://github.com/cosmos/ledger-go/pull/4 is merged

## Steps to Reproduce

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned
- [ ] Is a spike necessary to map out how the issue should be approached?

**Comment by gjermundgaraba** [comment]
[#4](https://github.com/cosmos/ledger-go/pull/4) was just merged. Do we need to wait for it to cut a release before we could make a PR for this (I would be happy to update this, test it and make a PR)?

**Comment by okwme** [comment]
great to hear!
i've just asked in the PR whether there will be a release or not, their last one wasn't since 2019.
would be amazing if you were able to update, test and make a new PR based on their reply @bjaanes thank you so much for the contribution!

**Comment by gjermundgaraba** [comment]
I just saw 0.9.3 was cut, so I created a PR to update that. Let me know if want any changes.
